src/main/java/org/isf/patvac/gui/PatVacEdit.java

```diff
@@ -513,27 +536,29 @@ private void setPatient(Patient selectedPatient) {
 	 * 
 	 * @return patientComboBox (JComboBox)
 	 */
-	private JComboBox getPatientComboBox(String regExp) {
-
+	private JComboBox<Object> getPatientComboBox(String regExp) {
+		if (patientComboBox == null) {
+			patientComboBox = new JComboBox<Object>();
+		}
+		patientComboBox.addItem(MessageBundle.getMessage("angal.patvac.selectapatient"));
 		Patient patSelected = null;
-		PatientBrowserManager patBrowser = Context.getApplicationContext().getBean(PatientBrowserManager.class);
 
 		if (GeneralData.ENHANCEDSEARCH){
 			try {
-				pat = patBrowser.getPatientsByOneOfFieldsLike(regExp);
+				patientList = patientManager.getPatientsByOneOfFieldsLike(regExp);
```

Comment:
should probably be spaces around the "catch"

Reply:
Done. Found only one...